docs/modules/ROOT/partials/runbooks/connect_to_postgres.adoc

```diff
@@ -2,10 +2,12 @@ Connect with the billing database.
 +
 [source,bash]
 ----
-kubectl -n appuio-reporting port-forward svc/reporting-db 5432 &
+INSTANCE_NS=$(kubectl -n appuio-cloud-reporting get vshnpostgresql cloud-reporting-db-prod -o jsonpath='{.status.instanceNamespace}')
```

Comment:
This is quite VSHN-specific, but that's probably fine, since the previous iteration already used a config which wasn't necessarily universally applicable.